### PR TITLE
Cleanup of conn.lib

### PR DESCRIPTION
**Pull request by jkriege2**
- moved PAL-symbols to new pld_pal.lib (these will need overhaul in a separate PR)
- renamed PCI&ISA-bus "connectors" in a common style, updated descriptions
- unified connector-naming style CONN->Conn, BUS->Bus, ...
- moved remaining IC (PCI9030-PQFP176) to interface.lib + added FPFilters (symbol also needs overhaul in a separated PR)

This is done as a preparation (in parts) for the proposed changes by @poeschlr there: https://github.com/KiCad/kicad-library/issues/1451

PS: This is not about conforming to KLC, and I will add additional PRs for cleaning up the KLC-issues!

### CODE REVIEW

**Comment by evanshultz** [comment]
@jkriege2 
Would you want to add FPfilters to all `Conn_xxXxx` parts here, or wait until later? Currently, it seems there's many missing, or the footprints don't exist, or some other issue. I would think at a minimum, each of these parts could at least have `Conn*`, `Pin*`, and `Socket*` for the FPfilter to at least get rid of most other footprints which aren't close to a connector.

Perhaps it's best to do this later once we have things more settled?

**Comment by jkriege2** [comment]
I'm not sure about FPFilters: There are so many matching connectors and no naming convention on the side of connectors. therefore the best would be (IMHO) to either filter by Library, or simply omitting the FPFilter.

I would like to merge this small PR now and then decide later, if we want FPFilters, or other changes ...
What do you think?

Comments on the other changes?

JAN

**Comment by evanshultz** [comment]
Many of them have an FPfilter now. At least in the fullness of time, I think it's helpful to have some structure here so that connectors are filtered by default.

But reviewing the other changes.
-I really like the camelcase. Much more readable. And the renaming of some of the computer bus connectors is much more salient.
-Is there a typo here?
image
-Should these be camelcase too?
image
image
-Regarding the libraries added and parts moved, I'll let Oliver weigh in on that. Have the new libraries been all settled?

**Comment by poeschlr** [comment]
My script will be capable to generate symbols for single and dual row connectors. Including Conn_Male and Conn_Female if we decide we want to keep them.
(Why do we want male and female variants for single row connectors but not for dual row connectors.)

I will add footprint filters of the style Connector\*:\*_[number of rows]x\* for multi row connectors and Connector\*:\* for single row connectors. (currently most if not all single row connectors do not include the number of rows in their name.)
According to https://github.com/KiCad/kicad-library/issues/1170 this should work for nightly builds and be ignored in stable. (the underline is necessary to filer the number of rows instead of the number of pins.)

In other words: Within the next few days the symbols will be replaced by scrip generated variants. So don't use too much time in this pull request for these symbols.

**Comment by jkriege2** [comment]
OK,

then I will do the smaller Camelcase-Renames and we can go from there. @poeschlr please use the Camelcase-naming?

Best,
JAN

**Comment by jkriege2** [comment]
OK, I fixed the typos/missing renames above.
The part-moving (that strange PCI-controller) should be fine by our current standards, so we can move it on, when we do the restructuring. The PAL-devices ... mmhhh ... I think we didn't decide on a general programmable logic lib, so I took a name that was close-enough and starts with `logic_` ;-) Also these symbols are of VERY low quality. If I find some time, I might draw up proper PAL/GAL symbols ... at least a few, so we have something. I think these are not really used anymore today?

Best,
JAN

**Comment by poeschlr** [comment]
I will use camel case naming for the connectors I create.
What about male and female variants? Keep or delete. (If we keep them should there also be such symbols for multi row connectors?)

**Comment by poeschlr** [comment]
Can't edit my comment on mobile. 
I vote for deletion but would include a note in the readme that there is a script capable to generate such symbols.

**Comment by jkriege2** [comment]
I would keep them ... I find them a neat extension, especially since you can pair the variants in the lib, which can be handy sometimes, BUT I think the limited number of single-row MALE/FEMALEs we have now is sufficient. I think that was also decided, when we discussed that particular PR: https://github.com/KiCad/kicad-library/pull/816
... in the end we also have special connector symbols for other cases (screw terminals, wall-plugs/sockest/MINI-DIN,SUB-D,coaxial...) basically everything, where we want the possibility to say explicitly what we mean in the schematic, but we limited also these to reasonable amounts (I think).

JAN

**Comment by poeschlr** [comment]
@evanshultz anything else that needs to be done here?
I will open my pull request as soon as this has been merged.

**Comment by evanshultz** [comment]
If the libraries are OK, it's fine with me. I only took a quick look and saw the typos. Thanks for checking!

**Comment by jkriege2** [comment]
So can this be merged? @evanshultz @poeschlr

**Comment by poeschlr** [comment]
I had another closer look. I can't find any problems with this PR. Thanks for your work.
